app/views.py

```diff
@@ -11,112 +20,85 @@ def home():
 
     return render_template("index.html")
 
+#TODO ADD ROLES/ ADD ADMIN PANEL ?
 @app.route("/show_users")
 def users():
-    conn = database_context.connect()
+    
+    users = UsersRepository.getUsers()
 
-    c = conn.cursor()
+    info = []
```

Comment:
are there lambdas in python? something that would let you solve problem below, by typing info = <some lambda on users> directly? If yes, then you should start rewriting fors to lambdas in next pr

Reply:
done